momentum/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To do list</title>
+    <style>
+      /* 간단한 스타일링 */
+      body {
+        font-family: Arial, sans-serif;
+        margin: 20px;
+      }
+      input[type='text'] {
+        padding: 8px;
+        width: 200px;
+      }
+      button {
+        padding: 8px;
+        margin-left: 5px;
+      }
+      ul {
+        list-style-type: none;
+        padding-left: 0;
+      }
+      li {
+        padding: 5px 0;
+        display: flex;
+        align-items: center;
+      }
+      li.completed {
+        text-decoration: line-through;
+        color: gray;
+      }

```

Comment:
외부 css 파일을 적용시키기 위해서는 
```suggestion
      }
 <link rel="stylesheet" href="style.css" />
```
이런식으로 html의 header내에 스타일을 링크해주면 됩니다.